xml/System.Text/Rune.xml

```diff
@@ -643,10 +671,20 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <Parameter Name="value" Type="System.Text.Rune" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The <see cref="T:System.Text.Rune" /> to evaluate.</param>
+        <summary>Indicates whether the specified <see cref="T:System.Text.Rune" /> is categorized as a decimal digit.</summary>
+        <returns><see langword="true" /> if <paramref name="value" /> is a decimal digit; otherwise <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method is equivalent to querying <xref:System.Text.Rune.GetUnicodeCategory(System.Text.Rune)> and comparing the result against <xref:System.Globalization.UnicodeCategory.DecimalDigitNumber>.
+
+See <xref:System.Char.IsDigit> for more information.
```

Comment:
In case the above is not possible, here's the suggestion.
```suggestion
See <xref:System.Char.IsDigit?displayProperty=nameWithType> for more information.
```